include/veriblock/storage/endorsement_repository_inmem.hpp

```diff
@@ -148,6 +148,10 @@ struct EndorsementRepositoryInmem : public EndorsementRepository<Endorsement> {
     return ret;
   }
 
+  std::unique_ptr<EndorsementWriteBatch<Endorsement>> newBatch() {
+    return nullptr;
```

Comment:
?

Reply:
We dont have inmem WriteBatch realisation, and generally dont use it, so here it is a mock of this method.